Android | reduce minSdkVersion to 19 (#75)
ably/ably-flutter: README.md

[Support and Help](https://www.ably.io/support)
site to discuss privately.

## Supported Platforms

#### iOS

> iOS9 or newer

Reviewer: ```suggestion
iOS 9 or newer.
```

Author: Addressed: a23b04ef17039c4a68d05220ea76366e3e0e974e

#### Android

> SDKVersion 19 or newer.

Reviewer: API Level 19 (Android 4.4, KitKat) or newer.

Author: Addressed: a23b04ef17039c4a68d05220ea76366e3e0e974e

> _This project uses Java8 language features and uses [Desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring)

Reviewer: ```suggestion
_This project uses Java 8 language features, utilising [Desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring)
```

Author: Addressed: a23b04ef17039c4a68d05220ea76366e3e0e974e

> to support lower versions of android (< SDKVersion 24)_

Reviewer: ```suggestion
to support lower versions of the Android runtime (i.e. API Levels prior to 24)_
```

Author: Addressed: a23b04ef17039c4a68d05220ea76366e3e0e974e

> If your project needs support for lower SDK Version, Android Gradle Plugin 4.0.0+ must be used.

Reviewer: Lower than what? Can you make this more explicit please.

Author: Addressed: a23b04ef17039c4a68d05220ea76366e3e0e974e

You might also need to upgrade [gradle distribution](https://developer.android.com/studio/releases/gradle-plugin#updating-plugin) accordingly.

> 

Reviewer: Please avoid multiple new lines. One is always enough. 😁

Author: Addressed: a23b04ef17039c4a68d05220ea76366e3e0e974e
